lib/Document.php

```diff
@@ -112,10 +127,23 @@ public function download(?bool $postProcess = true, ?string $encoding = null): s
         // typically ISO-8859-1 encoded, which messes up the data when we read it directly via
         // SimpleXML or as a plain TAB/CSV, but the original encoding is required to parse XLSX
         // and PDF reports.
+        // If encoding is not provided try to automatically detect the encoding from the http response; default is UTF-8
         if (!($this->contentType === ContentType::XLSX || $this->contentType === ContentType::PDF)) {
             if (!is_null($encoding) && !in_array(strtoupper($encoding), mb_list_encodings())) {
                 $encoding = null;
             }
+            else
+            {
+                $encodings = array('UTF-8');
```

Comment:
Please change this to `$encodings = ['UTF-8'];`

Reply:
Done in last commit